src/test/items/quick_powder.test.ts

```diff
@@ -37,29 +37,29 @@ describe("Items - Quick Powder", () => {
 
     const partyMember = game.scene.getParty()[0];
 
-    // Checking consoe log to make sure Quick Powder is applied when getBattleStat (with the appropriate stat) is called
-    partyMember.getBattleStat(Stat.DEF);
+    // Checking consoe log to make sure Quick Powder is applied when getEffectiveStat (with the appropriate stat) is called
```

Comment:
```suggestion
    // Checking console log to make sure Quick Powder is applied when getEffectiveStat (with the appropriate stat) is called
```
NIT: Minor correction